integrations/src/test/resources/io/jenkins/plugins/casc/JiraTest.yml

```diff
@@ -1,7 +1,6 @@
 unclassified:
-  jiraProjectProperty:
+  jiraGlobalConfiguration:
     sites:
-      url: "http://jira.codehaus.org/"
-# see https://issues.jenkins.io/browse/JENKINS-52906
-#      - url: "http://jira.codehaus.org/"
-#      - url: "http://issues.jenkins.io/"
+      - url: "http://jira.codehaus.org/"
+      # see https://issues.jenkins.io/browse/JENKINS-52906
```

Comment:
```suggestion
```
This can be removed

Reply:
I try to keep it in order to reference historical regressions. But whatever you prefer @casz